CONTRIBUTING.md

```diff
@@ -29,26 +31,119 @@ The [PyScript Code of Conduct](https://github.com/pyscript/governance/blob/main/
 
 # Contributing
 
-## Reporting bugs
+## Opening an Issue
+
+If you encounter a bug, have a feature proposal, or a question, please open an issue on the PyScript repository and select. When opening an issue, make sure to include the following information:
```

Comment:
Do we want to mention that some things people should open discussions instead? Or will it be up to the maintainers to create discussions if appropriate?

Reply:
Yeah, I'll change it to 

> If you encounter a bug, have a feature proposal or have a question, please open an issue on the PyScript repository and select the right category. Feature proposals and questions will be opened as a `Discussion` to encourage discussions around the topic, while bugs are filed under traditional Github Issues. When opening an issue, make sure to include the following information: